back/Gemfile.lock

```diff
@@ -1134,7 +1139,7 @@ GEM
     zeitwerk (2.6.0)
 
 PLATFORMS
-  ruby
```

Comment:
weird.. I wonder where this is coming from and what are the implications.

Reply:
no idea, im running `bundle install` from within the container and getting that lock file as result